front/src/config/i18n/fr.json

```diff
@@ -187,6 +187,7 @@
     "editDashboardCancelButton": "Annuler",
     "editDashboardDeleteButton": "Supprimer",
     "editDashboardSaveButton": "Sauvegarder",
+    "editDashboardDeleteText": "Etes-vous sur de vouloir supprimer ce dashboard ?",
```

Comment:
A few typos + "dashboard" => "tableau de bord" in french :) 

> Êtes-vous sûr de vouloir supprimer ce tableau de bord ?

Reply:
Oups :(
Fixed now